src/deepsparse/yolo/schemas.py

```diff
@@ -51,9 +51,9 @@ class YOLOInput(BaseModel):
     )
 
     @classmethod
-    def from_files(cls, files: List[str], **kwargs) -> "YOLOInput":
+    def from_files(cls, files: Iterable[TextIO], **kwargs) -> "YOLOInput":
         """
-        :param files: list of file paths to create YOLOInput from
+        :param files: Iterable of file pointers to create YOLOInput from
```

Comment:
why don't we need any extra processing here?

Reply:
Good catch Ben, forgot to push the commit, it's pushed now